code/game/objects/items/devices/flashlight.dm

```diff
@@ -216,6 +216,9 @@
 	else
 		update_brightness()
 
+/obj/item/device/flashlight/flare/ground_act(var/obj/item/weapon/reagent_containers/RC)
```

Comment:
Yeah, everything goes fucking haywire if you don't painstakingly define any and all arguments exactly how they are in the parent
Well it can, at least